### ISSUE TITLE
xmod exploit possible for Rubber with NotEnoughWands "Moving wand"

### ISSUE THREAD

**Issue by xenoflot** [TechReborn/TechReborn#889]
Greetings!

tl;dr If moved with a NotEnoughWands "Moving Wand", (or it seems, just an axe) rubber blocks will reinitialize, potentially creating rubber sap spots were there were none.

---

It might be considered cheaty, but it's awfully convenient... I used a moving wand to grab all the rubberwood blocks from my naturally grown rubber trees and created a wall of logs where the sap spots are all facing the same direction. I just stacked the blocks w/o sap spots in a big pile. Then I went to bed (IRL!)

* I noticed when placing the sappy blocks that I'd have to pick and place several times in order to get the sap spot facing towards me. This suggests that the facing of the sap spot is being determined randomly when placed each time.
* After logging in again this morning, a good number of the blocks in my "pile of sapless blocks" have sprouted sap spots!
* Using an axe, I just broke the blocks from the pile and created 2 new piles. One of blocks that were sapless and one of blocks that had sap. Some of these "dead" blocks are now regrowing sap! In the immortal words of Dr Ian Malcolm, life uh... finds a way, but I worry that my other crops will start a religion based on rubbery resurrection and start demanding tax exemptions for their greenhouses.

Availability of rubber is obviously a minor issue but IC2 originally went to some lengths to ensure that rubber tree sap log blocks could not be moved or rotated. Since my understanding is that TR is aiming for the GregTech Experience, you might want to put some extra logic in the log blocks can't be moved, or at least don't reinitialize if broken and re-placed. 

TR 1.10.2-1.4.16.133
rbcore 1.10.2-2.8.5.79
forge 2185

Cheers!

**Comment by Prospector** [comment]
I'm pretty sure TR has no such thing as sap spots. They just pop up whereever. Which IMO is fine. Rubber isn't exactpy supposed to be challenging to get. I'll leave this open for a bit for a debate if anyone thinks it should be otherwise.
